### PR TITLE
Implemented decay persistency and removed unused assets

### PR DESCRIPTION
**Pull request by DaanTacoSchool**
Removed unnecesary assets that i imported. Only the ones used and the icons are still in it. All assets that are not in the "Custom" or "Scenes" folders,  arent mine.

Also implemented persistance for dates from which is calculated how much the meters have dropped in the oftime.

### CODE REVIEW

**Comment by DaanTacoSchool** [comment]
note; DO NOT DELETE BRANCH

File: SeriousGame/Assets/Custom/Scripts/AdjustDataScript.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System.Collections;
+﻿    using System.Collections;
```
Comment (Jensderond) [inline SeriousGame/Assets/Custom/Scripts/AdjustDataScript.cs]: What happened here?

File: SeriousGame/Assets/Custom/Scripts/GameController.cs [hunk 2]
```diff
@@ -50,17 +57,19 @@ private void OnDisable()
         {
             EnergyLevel = EnergyLevel,
             FoodLevel = FoodLevel,
-            WaterLevel = WaterLevel
-        };
+            WaterLevel = WaterLevel,
+            OldDate = System.DateTime.Now
+
+    };
```
Comment (Jensderond) [inline SeriousGame/Assets/Custom/Scripts/GameController.cs]: Indention gone wrong, please run: `Ctrl + K + D` (assuming u use VS 2017) to format the code.

**Comment by Jensderond** [comment]
You always have your branch locally so a delete wouldn't make a difference.
Added some notes to the changed scripts.
If you could fix those I you got my approval!

File: SeriousGame/Assets/Custom/Scripts/AdjustDataScript.cs (hunk 1 above)
Comment (DaanTacoSchool) [reply SeriousGame/Assets/Custom/Scripts/AdjustDataScript.cs]: autoformat probably

File: SeriousGame/Assets/Custom/Scripts/GameController.cs (hunk 2 above)
Comment (DaanTacoSchool) [reply SeriousGame/Assets/Custom/Scripts/GameController.cs]: will incorporate that in the next commit which will include content as well

**Comment by DaanTacoSchool** [comment]
updated with new commit

File: SeriousGame/Assets/Custom/Scripts/GameController.cs (hunk 2 above)
Comment (Jensderond) [reply SeriousGame/Assets/Custom/Scripts/GameController.cs]: Okay, will wait for the next commit. Please also do this for AdjustDataScript.cs as well. 👍

**Comment by DaanTacoSchool** [comment]
@Jensderond @semrekkers @koentjeh @cliffsestig @larsheesakkers1

**Pull request merged**